hic_qc.py

```diff
@@ -385,6 +390,19 @@ def update_mapped_pair_stats(self, a, b):
                 self.stats['pairs_greater_10k'] += 1
             if dist == 0:
                 self.stats['zero_dist_pairs'] += 1
+
+            ## Additional stats for dynamo table.
+            if 0 <= dist < 1e3:
+                self.stats['reads_spanning_up_to_1k'] += 1
+            elif 1e3 <= dist < 1e4:
```

Comment:
One minor note: since you have handled the case where dist is < 1e3 already, it would be enough to have this be `elif dist < 1e4`

Reply:
Thanks for looking! If I do this, though, the pair won't be counted if the distance is exactly 1e3, right?

Reply:
It would go in the "reads_spanning_up_to_10k" bucket, since the pair isn't less than 1k, but it is less than 10k.

Reply:
That makes sense! Thank you for clarifying.